kiwi_lint/similar_string.py

```diff
@@ -0,0 +1,70 @@
+import string
+
+import astroid
+from pylint.checkers import BaseChecker, utils
+from pylint.interfaces import IAstroidChecker
+from textdistance import levenshtein
+
+
+class SimilarStringChecker(BaseChecker):
+    __implements__ = (IAstroidChecker,)
+    name = "similar-string-checker"
+
+    msgs = {
+        "R5011": (
+            'A similar string to "%s" exists: "%s"',
```

Comment:
Let's change the message to `"%s" is similar to "%s". Try keeping them the same to reduce work for translators!`